includes/MirahezeMagicHooks.php

```diff
@@ -132,6 +132,27 @@ public static function onHtmlPageLinkRendererEnd( $linkRenderer, $target, $isKno
 		return true;
 	}
 
+	/**
+	 * Hard redirects all pages like Mh:Wiki:Page as global interwiki.
+	 */
+	public static function onInitializeArticleMaybeRedirect( $title, $request, &$ignoreRedirect, &$target, $article ) {
```

Comment:
This static function doesn't seem to be used?

Reply:
:I I knew I was forgetting something